src/sql/workbench/api/common/extHostModelViewDialog.ts

```diff
@@ -688,13 +714,13 @@ export class ExtHostModelViewDialog implements ExtHostModelViewDialogShape {
 		this._onClickCallbacks.set(handle, callback);
 	}
 
-	public createDialog(title: string, dialogName?: string, extension?: IExtensionDescription, isWide?: boolean): azdata.window.Dialog {
+	public createDialog(title: string, dialogName?: string, extension?: IExtensionDescription, width?: azdata.window.DialogWidth): azdata.window.Dialog {
 		let dialog = new DialogImpl(this, this._extHostModelView, this._extHostTaskManagement, extension);
 		if (dialogName) {
 			dialog.dialogName = dialogName;
 		}
 		dialog.title = title;
-		dialog.isWide = isWide;
+		dialog.width = width === undefined ? 'narrow' : width;
```

Comment:
FYI you can shorten this to `dialog.width = width ?? 'narrow'` since that'll still allow falsy values like 0.